docs/install.rst

```diff
@@ -1,3 +1,53 @@
+Dependencies
+============
+A **minimal installation** of Stingray requires the following dependencies:
+
++ astropy>=4.0
++ six
```

Comment:
Do we still need six if we no longer support Python2?

Reply:
Good catch. Eliminated from setup.cfg as well.